src/kernel/kmain.zig

```diff
@@ -64,9 +65,25 @@ export fn kmain(mb_info: *multiboot.multiboot_info_t, mb_magic: u32) void {
         vga.init();
         tty.init();
 
+        log.logInfo("Init heap\n", .{});
```

Comment:
I'm not really sure what I could test at runtime, since it isn't actually dependent on runtime. It can all be tested with a normal `test`, unless you can think of anything!

Reply:
If it can all be done using `test`, then will be fine :)

Reply:
Agreed :) I've added a heap.init function now.